test/scenarios/activegate/installation.go

```diff
@@ -3,18 +3,16 @@
 package activegate
 
 import (
-	"bytes"
 	"context"
 	"fmt"
-	"io"
 	"strings"
 	"testing"
 
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
 	"github.com/Dynatrace/dynatrace-operator/src/controllers/dynakube/activegate/consts"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/activegate"
```

Comment:
this package does not exist in you branch (or you did not commit them)
I created that package in my example at https://github.com/Dynatrace/dynatrace-operator/tree/syntetic-e2e
so we don't repeat ourselves

Reply:
I excluded it unintentionally. Correcting....